conf/messages

```diff
@@ -103,6 +103,15 @@ tax-identifiers.field_1                                      = Enter a tax ident
 tax-identifiers.field_2                                      = Enter the name of the country that issued the tax identifier
 tax-identifiers.no_identifier_link                           = I do not want to provide an identifier
 
+## Overseas Tax Identifier Page
+tax-identifier.title                                         = Does the business have an overseas tax identifier?
+tax-identifier.line_1                                        = We may use this number to help us identify your business. The tax identifier could be VAT registration number, Employee Identification Number (EIN) or any other identifier we could use to verify your business details.
```

Comment:
The line_1 is missing an 'a' in "The tax identifier could be VAT" compared to the original hint text

Reply:
done